src/main/java/gregtech/api/DualHandler.java

```diff
@@ -0,0 +1,128 @@
+package gregtech.api;
+
+import gregtech.api.capability.IMultipleTankHandler;
+
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.fluids.FluidStack;
+import net.minecraftforge.fluids.FluidTankInfo;
+import net.minecraftforge.fluids.IFluidTank;
+import net.minecraftforge.fluids.capability.IFluidTankProperties;
+import net.minecraftforge.items.IItemHandlerModifiable;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+
+public class DualHandler implements IItemHandlerModifiable, IFluidTank, IMultipleTankHandler {
```

Comment:
Should this implement `IFluidTank`? What about if a dual handler has multiple fluid tanks?

Reply:
A problem that I ran into is that the single-tank fluid hatches expect `List<? extends IFluidTank>` for `registerAbilities()`. there's `getTanks()` from the dual handler, but the item handler would be lost with that.